src/style.css

```diff
@@ -1 +1,111 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc)
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --border-bars: #E1E7ED;
+  --bg-page: #FFF;
+  --shadow-bars: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  --shadod-bars-hover: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+  --text-color: #3D4E61;
```

Comment:
Create a variable with root selector

Reply:
not fixed https://github.com/mate-academy/layout_search-bar-airbnb/pull/3817/files#r1407405161